src/itables/datatables_format.py

```diff
@@ -110,7 +110,7 @@ def datatables_rows(df, count=None, warn_on_unexpected_types=False, pure_json=Fa
         )
     except AttributeError:
         # Polars DataFrame
-        data = list(df.iter_rows())
+        data = list(df.rows())
```

Comment:
just `data = df.rows()`, you don't need `list`

Reply:
of course, sorry